### PR TITLE
feat(manager): implement MutableMapping-related methods in class "Draft"

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-290mf70 实现 Draft MutableMapping 相关接口用于管理 Sheet](https://app.clickup.com/t/290mf70)

File: graviti/manager/draft.py [hunk 1]
```diff
@@ -49,27 +68,98 @@ def __init__(
...
 
-        Return:
+        Returns:
             The keys in dict.
 
         """
+        return self._get_data().keys()
+
+    def clear(self) -> None:
+        """Remove all items from the mutable mapping object."""
```
Comment (Lee-000) [inline graviti/manager/draft.py:95]: `mutable mapping` -> `Draft`?

File: graviti/manager/draft.py [hunk 2]
```diff
@@ -49,27 +68,98 @@ def __init__(
         pass
 
     def __len__(self) -> int:
-        pass
+        return self._get_data().__len__()
```
Comment (Lee-000) [inline graviti/manager/draft.py:71]: Is it necessary to write all the mapping methods into a mixin class?

File: graviti/manager/base.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+"""Basic methods of some manager classes."""
+
+from typing import Any, Dict
+
+from tensorbay.dataset import Notes, RemoteData
+from tensorbay.label import Catalog
+from tensorbay.utility import URL
+
+from graviti.client import get_catalog, get_notes, list_data_details, list_segments
```
Comment (linjiX) [inline graviti/manager/base.py:14]: We still use the old OpenAPIv1 here, shall we update them to v2?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/manager/base.py (hunk 3 above)
Comment (graczhual) [reply graviti/manager/base.py]: After the openapi is updated, we will push another pr to update the `api` and the `params in Sheets`. Now, Draft can not use `self.access_key`

File: graviti/manager/sheets.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+class Sheets(MutableMapping[str, DataFrame]):
+    """The basic structure of the Graviti sheets."""
+
+    _data: Dict[str, DataFrame]
+    _dataset_id: str
+    access_key: str
+    url: str
+    commit_id: str
+
```
Comment (linjiX) [inline graviti/manager/sheets.py:30]: do we need a `__init__` method to init these members?

File: graviti/manager/draft.py [hunk 5]
```diff
@@ -5,17 +5,17 @@
...
 from graviti.manager.lazy import PagingList
+from graviti.manager.sheets import Sheets
 from graviti.openapi import create_draft, get_draft, list_drafts
 
 if TYPE_CHECKING:
     from graviti.manager.dataset import Dataset
 
 
-class Draft(MutableMapping[str, DataFrame]):
+class Draft(Sheets):
```
Comment (Lee-000) [inline graviti/manager/draft.py:18]: Why `Draft` does not inherit `AttrsMixin, ReprMixin`?

File: graviti/manager/sheets.py (hunk 4 above)
Comment (graczhual) [reply graviti/manager/sheets.py:30]: We will do that after the

File: graviti/manager/sheets.py [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+        return self._get_data().__getitem__(key)
+
+    def __iter__(self) -> Iterator[str]:
+        return self._get_data().__iter__()
+
+    def __setitem__(self, key: str, value: DataFrame) -> None:
+        self._get_data().__setitem__(key, value)
+
+    def __delitem__(self, key: str) -> None:
+        self._get_data().__delitem__(key)
```
[the remark this answers is not in the document]
Comment (graczhual) [reply graviti/manager/sheets.py:44]: Yes. Dataset can also change, but it can not upload

File: graviti/manager/draft.py (hunk 5 above)
Comment (graczhual) [reply graviti/manager/draft.py:18]: we will add repr in another pr

**Review by linjiX — APPROVED** [review]

**Review by Lee-000 — APPROVED** [review]

**Pull request merged**